faq/index.html

```diff
@@ -87,10 +88,27 @@ <h6 class="arrow-r">Where is the documentation?</h6>
                     <li class="submenu">
                         <h6 class="arrow-r">What is the roadmap?</h6>
                         <div class="submenu-content">
-                            <p>The roadmap is being tracked in GitHub <a target="_blank" href="{{ site.data.config.milestones_link }}">milestones</a>. Please refer to the respective milestones in the gRPC, gRPC-java and gRPC-go repositories under the GitHub gRPC org.</p>
+                            <p>The gRPC project has an RFC process, through which new features are designed and approved for implementation. They are tracked in <a target="_blank" href="https://github.com/grpc/proposal"> this repository</a>. </p>
+                        </div>
+                    </li>
+                    <li class="submenu">
+                        <h6 class="arrow-r">What is the latest gRPC Version?</h6>
+                        <div class="submenu-content">
+                            <p>The latest release branch is {{site.data.config.grpc_release_branch}}</p>
```

Comment:
Mention that the different repos may be slightly out-of-sync around the time of a release?